framework/src/modules/chain/blocks/block_version.js

```diff
@@ -49,7 +49,32 @@ const isValid = (version, height, exceptions = {}) => {
 	return Number(exceptionVersion) === version;
 };
 
+const getBlockVersion = (height, exceptions = {}) => {
+	if (height === undefined || Object.entries(exceptions).length === 0) {
```

Comment:
`if(height === undefined || !Object.keys(exceptions))` I think is easier to read and also  `Object.entries` is very slow

Reply:
`Object.keys` works better thanks! Need the explicit check tho.